src/components/ChoiceFieldGroup/ChoiceFieldGroup.hbs

```diff
@@ -1,9 +1,11 @@
 <!-- Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information. -->
-<div class="ms-ChoiceFieldGroup">
+<div class="ms-ChoiceFieldGroup" id="{{props.id}}">
   <div class="ms-ChoiceFieldGroup-title">
     <label class="ms-Label {{#if props.required}} is-required {{/if}}">{{props.title}}</label>
   </div>
+  <ul class="ms-ChoiceFieldGroup-list">
```

Comment:
Add `role=radiogroup` to indicate that this element contains all of the `radio` elements.